### DISCUSSION TITLE
I have a problem with the start menu

### DISCUSSION THREAD

**Discussion by dreamaker2107** [Open-Shell/Open-Shell-Menu#1844] [Q&A] [answered]
I bought a new pc recently and they apparenlty added the open shell for customization purposes. Well, it has a windows 8 like menu and i don't know how to change it since the settings menu is different than the one shown in the guides and tutorials online. For example, it says to go to the "Controls" tab and change the "left click opens" and "windows key opens" but my open shell settings don't even have a "Controls" tab. Any help is much appreciated since i really like the classic windows 10 start manu and i am really comfortable with it, so i don't want to change it!
![openshellsettings](https://github.com/Open-Shell/Open-Shell-Menu/assets/139267997/6ebd4b7f-c40f-42f6-a67e-ef18a54f52a6)

**Comment by ge0rdi** [comment, marked as answer]
> the settings menu is different than the one shown in the guides and tutorials online

According to the screenshot you are looking at `Classic Explorer Settings` (those are settings for our Explorer extension).
You need to open `Open-Shell Menu Settings` shortcut from Start menu.

>  i really like the classic windows 10 start manu and i am really comfortable with it, so i don't want to change it!

You can just uninstall `Open-Shell` if you want original Windows start menu and don't want any customizations that `Open-Shell` provides.

**Reply by dreamaker2107** [reply]
>According to the screenshot you are looking at Classic Explorer Settings (those are settings for our Explorer extension).
You need to open Open-Shell Menu Settings shortcut from Start menu.

Thanks a lot! I thought that those were the settings for the customization.... Thanks again!!
